### ISSUE TITLE
SelectOneMenu: Screen reader skips empty entry 

### ISSUE THREAD

**Issue by Joern2026** [primefaces/primefaces#15221]
### Describe the bug

NVDA does not output any text for the empty entry in the SelectOneMenu. There is no way to define a text (e.g.  aria-label).

<img width="724" height="395" alt="Image" src="https://github.com/user-attachments/assets/297c52d0-777a-484f-b45b-34f2cdf30f2e" />

### Reproducer

[primefaces-test-15.0.18.zip](https://github.com/user-attachments/files/32055756/primefaces-test-15.0.18.zip)

### Expected behavior

Option to define a (local) text or use an application-wide text representing "Empty".

### PrimeFaces edition

Community

### PrimeFaces version

15.0.18

### JSF implementation

Mojarra

### JSF version

4.0

### Java version

21

### Browser(s)

Chrome

**Comment by Joern2026** [comment]
Hello @melloware,
thank you very much for your quick response and fixing these issues. Is it possible to merge the changes also in the PF 15 branch so we can use it with the next release?

Thank you very much and best regards
Jörn

**Comment by melloware** [comment]
Sure I can backport it to 15

**Comment by melloware** [comment]
@Joern2026 i am backporting to 15.X but since its deployed here https://showcase.primefaces.org/ui/input/oneMenu.xhtml?jfwid=98744 can you test the fix with  your screenreader please?

**Comment by Joern2026** [comment]
@melloware thank you very much. I will test it tomorrow. Regarding the showcase, does it makes sense to add an example like in my reproducer? 3 items and one of them empty? The examples in the showcase are all a little bit different. I think properties like hideNoSelectionOption, noSelectionOption and itemLabels unequal empty string will influence the behavior.

**Comment by melloware** [comment]
ok showcase updated

<img width="1539" height="540" alt="Image" src="https://github.com/user-attachments/assets/3e41bbd4-edb2-4458-b234-003d78300e97" />

**Comment by melloware** [comment]
@Joern2026 were you able to test it?

**Comment by Joern2026** [comment]
@melloware I tested it and saw following behavior:
#15221 seems to me, that it works now
#15222 Because of the explaination "both in the options list and on the closed menu once it is selected" I expected, that "Not Selected" is the output while tabbing into after the empty entry was selected before. But except "with placeholder" in all examples I think it doesn't work. From first example the output is "Empty Item Label  Kombinationsfeld  reduziert". Consider the two spaces. 
Output of "Placeholder" example: "Empty Item Label with Placeholder  Kombinationsfeld  Choose One  reduziert".
